.travis.yml

```diff
@@ -8,11 +8,12 @@ addons:
       - cmake
       - gfortran
       - libnetcdf-dev
+      - libnetcdff-dev
```

Comment:
Did you mean to re-introduce this package here? (Since there was a commit explicitly removing it)

Reply:
I added this package by mistake, and afterwards removed it. On Debian, the fortran headers are located in it. On Ubuntu everything is included in `libnetcdf-dev`.